web/pages/[username]/[contractSlug].tsx

```diff
@@ -63,8 +66,18 @@ export async function getStaticPropz(props: {
   const contract = (await getContractFromSlug(contractSlug)) || null
   const contractId = contract?.id
   const bets = contractId
-    ? await listBets({ contractId, limit: 4000, ...CONTRACT_BET_FILTER })
+    ? await listBets({
```

Comment:
With no limit, seems like we could still hit the Vercel size limit and have static props error out.

Maybe just increase the limit. 10k or 12k?

Reply:
good idea